x-pack/plugins/security_solution/public/resolver/view/graph_controls.test.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Simulator } from '../test_utilities/simulator';
+import { noAncestorsTwoChildren } from '../data_access_layer/mocks/no_ancestors_two_children';
+import { nudgeAnimationDuration } from '../store/camera/scaling_constants';
+import '../test_utilities/extend_jest';
+
+describe('graph controls', () => {
+  let simulator: Simulator;
+  let originEntityID: string;
+  const resolverComponentInstanceID = 'graph-controls-test';
+
+  beforeEach(async () => {
+    const {
+      metadata: { databaseDocumentID, entityIDs },
+      dataAccessLayer,
+    } = noAncestorsTwoChildren();
+
+    simulator = new Simulator({
+      dataAccessLayer,
+      databaseDocumentID,
+      resolverComponentInstanceID,
+    });
+    originEntityID = entityIDs.origin;
+  });
+
+  it('should load graph controls', () => {
+    expect(simulator.graphControlElement().length).toBe(1);
```

Comment:
This test might not be needed. The existence of a wrapper element isn't important to the AC of any feature and its specific to the implementation details. Maybe check that the 4 buttons exist instead?

Reply:
Yea, I checked for the existence of all the functional buttons instead